packages/devtools_app/assets/scripts/inspector_polyfill_script.dart

```diff
@@ -94,43 +105,45 @@ String addServiceExtensions() {
         return entry;
       }
     }
-    return null;
+    throw 'Enum value $name not found';
   }
 
   Future<Map<String, dynamic>> getLayoutExplorerNode(
       Map<String, String> parameters) {
-    final String id = parameters['id'];
-    final int subtreeDepth = int.parse(parameters['subtreeDepth']);
-    final String groupName = parameters['groupName'];
-    Map<String, Object> result = {};
+    final id = parameters['id'];
+    final subtreeDepth = int.parse(parameters['subtreeDepth'] as String);
+    final groupName = parameters['groupName'];
+    var result = <String, dynamic>{};
     final instance = WidgetInspectorService.instance;
     final root = instance.toObject(id);
     if (root == null) {
-      result = null;
-    } else {
-      result = instance._nodeToJson(
-        root,
-        InspectorSerializationDelegate(
-            groupName: groupName,
-            summaryTree: true,
-            subtreeDepth: subtreeDepth,
-            service: instance,
-            addAdditionalPropertiesCallback: (node, delegate) {
-              final Map<String, Object> additionalJson = <String, Object>{};
-              final Object value = node.value;
-              if (value is Element) {
-                final renderObject = value.renderObject;
+      return Future.value(<String, dynamic>{
+        'result': result,
+      });
+    }
+    result = instance._nodeToJson(
+      root as DiagnosticsNode,
+      InspectorSerializationDelegate(
+          groupName: groupName,
+          summaryTree: true,
+          subtreeDepth: subtreeDepth,
+          service: instance,
+          addAdditionalPropertiesCallback: (node, delegate) {
+            final Map<String, Object> additionalJson = <String, Object>{};
+            final Object value = node.value as Object;
+            if (value is Element) {
+              final renderObject = value.renderObject;
+              if (renderObject != null) {
```

Comment:
should this be checking that `renderObject.toDiagnosticsNode != null`?

Reply:
Is there a typo in your question? renderObject.toDiagnosticsNode is a Function.
Do you mean checking that the result of toDiagnosticsNode() is not null? It is now enforced as null safe in Flutter and has always been null safe by convention. The previous check was overly paranoid and triggered a lint error with null safety.